docs/installation/debian.rst

```diff
@@ -42,11 +42,18 @@ Install from apt.mopidy.com
 
 #. Add the archive's GPG key::
 
-       wget -q -O - https://apt.mopidy.com/mopidy.gpg | sudo apt-key add -
+       sudo mkdir -p /usr/local/share/keyrings
+       wget -q -O - https://apt.mopidy.com/mopidy.gpg | gpg --dearmor | \
+         sudo tee /usr/local/share/keyrings/mopidy-archive-keyring.gpg
 
 #. Add the APT repo to your package sources::
 
-       sudo wget -q -O /etc/apt/sources.list.d/mopidy.list https://apt.mopidy.com/buster.list
+       echo "\
+       deb [signed-by=/usr/local/share/keyrings/mopidy-archive-keyring.gpg] \
+       https://apt.mopidy.com/ buster main contrib non-free
+       deb-src [signed-by=/usr/local/share/keyrings/mopidy-archive-keyring.gpg] \
+       https://apt.mopidy.com/ buster main contrib non-free" | \
+         sudo tee /etc/apt/sources.list.d/mopidy.list
```

Comment:
Cant't we just update https://apt.mopidy.com/buster.list instead of moving this to the docs?

Reply:
I figured since the first command allows them to (inadvertently?) use some other directory, we'd be better off making them both explicit. And I have no means to update buster.list

Reply:
But I am all for keeping this in Buster.list, if we think that's acceptable.

Reply:
`buster.list` is managed in the `apt` repo. It is deployed on commit.

Reply:
Oh cool ok. So we'll go for that then?